app/src/main/java/org/mozilla/focus/navigation/ScreenNavigator.kt

```diff
@@ -192,38 +200,34 @@ open class ScreenNavigator(activity: HostActivity?) : DefaultLifecycleObserver {
     }
 
     interface Provider {
-        val screenNavigator: ScreenNavigator
+        fun getScreenNavigator(): ScreenNavigator?
```

Comment:
should it be nullable?

Reply:
It's not necessary to be nullable. I'll change it.